src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -105,10 +109,20 @@ private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException
         Long id = resultSet.getObject("id", Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
         Driver driver = new Driver();
         driver.setId(id);
         driver.setName(name);
         driver.setLicenseNumber(licenseNumber);
+        driver.setLogin(login);
+        driver.setPassword(password);
         return driver;
     }
+
+    public Optional<Driver> findByLogin(String login) {
+        List<Driver> drivers = getAll();
```

Comment:
you should create a separate query for this method that will be looking for driver by passed login